src/boot/bootctl.c

```diff
@@ -1014,24 +1014,25 @@ static int help(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s  [OPTIONS...] COMMAND ...\n"
-               "\n%sInstall/update/remove the systemd-boot EFI boot manager and list/select entries.%s\n"
-               "\nBoot Loader Commands:\n"
+        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+               "\n%5$sControl EFI firmware boot settings and manage boot loader.%6$s\n"
+               "\n%3$sGeneric EFI Firmware/Boot Loader Commands:%4$s\n"
                "  status              Show status of installed systemd-boot and EFI variables\n"
```

Comment:
If we're touching this... `status` works for other boot loaders too, right? So this shouldn't say "installed systemd-boot" but just "installed boot loader and EFI variables".

Reply:
yes, it does work for all. will fix.

(it does show some extra info for sd-boot though, as it can extract the version from it and so on)